sql-style-guide.md

```diff
@@ -0,0 +1,124 @@
+# SQL Style Guide
```

Comment:
Do we have a style/naming guidelines for create/drop index?

Reply:
No, not yet